syntax = "proto3";

package protos.kafka;
import "google/protobuf/wrappers.proto";
import "protos/kafka/base.proto";


// Publish on `processed_events` topic
message ProcessedEventPayloadKey {
}

message ProcessedEventPayloadValue {
  google.protobuf.UInt64Value account_id = 1;
  ProcessedIngestionEvent event = 2;
}
